plugins/account_history/include/golos/plugins/account_history/history_object.hpp

```diff
@@ -4,39 +4,52 @@
 #include <golos/protocol/operations.hpp>
 #include <golos/protocol/steem_operations.hpp>
 #include <chainbase/chainbase.hpp>
-
+#include <golos/chain/index.hpp>
 
 #include <golos/chain/steem_object_types.hpp>
 #include <golos/chain/witness_objects.hpp>
 
 #include <boost/multi_index/composite_key.hpp>
 #include <golos/plugins/account_history/plugin.hpp>
 
+#ifndef TAG_SPACE_ID
+#define TAG_SPACE_ID 12
```

Comment:
Rename to `ACCOUNT_HISTORY_SPACE_ID`

Reply:
There is a ACCOUNT_HISTORY_SPACE_ID in plugin.hpp with number 5. What should I do?

Reply:
Well. I removed that from plugin.hpp and saved in history_object.hpp